lib/email_api/emailer.py

```diff
@@ -73,34 +73,33 @@ def send_emails(self, emails: List[EmailParams]):
                 server.sendmail(
                     email_params.email_from,
                     send_to,
-                    self._build_email(email_params).as_string(),
+                    self.build_email_header(email_params).as_string(),
                 )
 
             logger.info(
                 "sending complete - time elapsed: %s seconds", time.time() - start
             )
 
-    def _build_email(self, email_params: EmailParams) -> MIMEMultipart:
+    def build_email_header(self, email_params: EmailParams) -> MIMEMultipart:
```

Comment:
Why did you change this? 
Technically this is not just building email header since it calls `build_email_body` in the method

Reply:
Originally I wanted to split it up, but I've reverted it and called it `build_email` (so we're not going in via protected)